internal/wizard/project/create.go

```diff
@@ -0,0 +1,291 @@
+				Options(options...).
+				Value(&config.Input.Openshift),
+			huh.NewConfirm().
+				Title("Do you want to define any other fields?").
+				Value(&additionalFields),
+		),
+	).WithTheme(themes.ThemeLagoon())
+
+	err = form3.Run()
+	if err != nil {
+		return nil, err
+	}
+
+	if additionalFields {
+		var fields []string
+		additionalFieldsOptions := []huh.Option[string]{
+			huh.NewOption("branches: Which branches should be deployed", "Branches"),
+			huh.NewOption("pullrequests: Which Pull Requests should be deployed", "PullRequests"),
+		}
+		if organizationConfirm {
+			additionalFieldsOptions = append(additionalFieldsOptions, huh.NewOption("owner", "AddOrgOwner"))
+		}
+		form4 := huh.NewForm(
+			huh.NewGroup(
+				huh.NewMultiSelect[string]().
+					Title("Select which fields you want to define").
+					Options(additionalFieldsOptions...).
+					Value(&fields),
+			),
+		).WithTheme(themes.ThemeLagoon())
+
+		err = form4.Run()
+		if err != nil {
+			return nil, err
+		}
+
+		if len(fields) == 0 {
+			return config, nil
+		}
+
+		var inputs []huh.Field
+		var devEnvLimit string
+		for _, field := range fields {
+			switch field {
+			case "StandbyProductionEnvironment":
+				inputs = append(inputs, huh.NewInput().Title(fmt.Sprintf("Enter value for '%s'", field)).Value(&config.Input.StandbyProductionEnvironment))
+			case "Branches":
+				inputs = append(inputs, huh.NewInput().Title(fmt.Sprintf("Enter value for '%s'", field)).Value(&config.Input.Branches))
+			case "PullRequests":
+				inputs = append(inputs, huh.NewInput().Title(fmt.Sprintf("Enter value for '%s'", field)).Value(&config.Input.PullRequests))
+			case "OpenshiftProjectPattern":
+				inputs = append(inputs, huh.NewInput().Title(fmt.Sprintf("Enter value for '%s'", field)).Value(&config.Input.OpenshiftProjectPattern))
+			case "Subfolder":
+				inputs = append(inputs, huh.NewInput().Title(fmt.Sprintf("Enter value for '%s'", field)).Value(&config.Input.Subfolder))
+			case "PrivateKey":
+				inputs = append(inputs, huh.NewInput().Title(fmt.Sprintf("Enter value for '%s'", field)).Value(&config.Input.PrivateKey))
+			case "BuildImage":
+				inputs = append(inputs, huh.NewInput().Title(fmt.Sprintf("Enter value for '%s'", field)).Value(&config.Input.BuildImage))
+			case "RouterPattern":
+				inputs = append(inputs, huh.NewInput().Title(fmt.Sprintf("Enter value for '%s'", field)).Value(&config.Input.RouterPattern))
+
+			case "DevelopmentEnvironmentsLimit":
+				inputs = append(inputs, huh.NewInput().
+					Title(fmt.Sprintf("Enter value for '%s'", field)).
+					Value(&devEnvLimit).
+					Validate(func(s string) error {
+						if s == "" {
+							return nil
+						}
+						_, err := strconv.ParseUint(s, 10, 64)
+						return err
+					}),
+				)
+
+			case "AddOrgOwner":
+				config.Input.AddOrgOwner = new(bool)
+				inputs = append(inputs, huh.NewConfirm().
+					Title(fmt.Sprintf("Enable '%s'?", field)).
+					Value(config.Input.AddOrgOwner))
+			}
```

Comment:
I'm not sure this is ever called?

I selected additional fields creating a project and then selected `owner` from the list, but I was not prompted for anything further

Reply:
Ok, it works now :\ weird

<img width="261" height="39" alt="image" src="https://github.com/user-attachments/assets/cf0d64e1-7bc7-4349-8782-a3c9527bbaac" />

The "select fields you want to define" shows this as `Add your account to the project group during creation`
But when it goes to the fields page, it being listed as `Enable 'AddOrgOwner'`, could this be changed to be more like the UI with "add my user to the project" or something. This way it also matches what was shown closer on the selection page?
<img width="253" height="53" alt="image" src="https://github.com/user-attachments/assets/60496d7f-a01e-427c-9856-23c9496887dd" />